 Fix recents janky layout while in RTL

the jank was introduced where the recents title was overlapping while in RTL mode

Steps to duplicate the janky layout

 - Make the mistake of merging the commit above
 - Go into Settings, developer options and enable RTL
 - Open recents

diff --git a/packages/SystemUI/src/com/android/systemui/recents/views/TaskViewHeader.java b/packages/SystemUI/src/com/android/systemui/recents/views/TaskViewHeader.java
index 007f6ee..ce538fd 100644
--- a/packages/SystemUI/src/com/android/systemui/recents/views/TaskViewHeader.java
+++ b/packages/SystemUI/src/com/android/systemui/recents/views/TaskViewHeader.java
@@ -286,6 +286,7 @@
                 ((secondaryButton != null ? mHeaderBarHeight : 0)
                 + (tertiaryButton != null ? mHeaderBarHeight : 0)));
         title.setLayoutParams(lp);
+        title.setTextDirection(View.TEXT_DIRECTION_LOCALE);
         if (secondaryButton != null) {
             lp = new FrameLayout.LayoutParams(mHeaderBarHeight, mHeaderBarHeight, Gravity.END);
             lp.setMarginEnd(mHeaderBarHeight);
